lang/po/Firmware_cs.po

```diff
@@ -935,7 +946,7 @@ msgstr ""
 #. MSG_IMPROVE_BED_OFFSET_AND_SKEW_LINE1 c=20 r=4
 #: ../../Firmware/mesh_bed_calibration.cpp:2530
 msgid "Improving bed calibration point"
-msgstr ""
+msgstr "Zlepsovani kalibracniho bodu"
```

Comment:
Upresnovani

Reply:
changing to `Upresnovani kalibracniho bodu`